code/src/components/Header.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Header = () => {
+    return (
+        <a href='https://open.spotify.com/genre/new-releases-page' rel="noopener noreferrer"><h1>New albums & singles</h1></a>
+)}
+
+export default Header;
```

Comment:
This component makes a link out of the entire div that wraps the header, which feels a bit risky when navigating the page that you will accidentally click this big link. If you wanted to only have the actual text be a link you could make the h1 into display: inline, then move the padding and border styling to the 'Header' div.